scripts/pinecone_setup.py

```
@@ -0,0 +1,342 @@
+                print(f"⚠️  Index '{index_name}' already exists!")
+                response = input("\nDo you want to delete and recreate it? (yes/no): ")
+                if response.lower() != 'yes':
+                    print("❌ Index creation cancelled.")
+                    return False
+
+                print(f"🗑️  Deleting existing index '{index_name}'...")
+                self.pc.delete_index(index_name)
+                print("✅ Index deleted.")
+
+            # Create index
+            print(f"\n📝 Creating index with configuration:")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)